Sources/Interaction/Manipulators/MouseCameraTrackballRotateManipulator/index.js

```diff
@@ -36,11 +36,12 @@ function vtkMouseCameraTrackballRotateManipulator(publicAPI, model) {
     mat4.identity(trans);
 
     const { center, rotationFactor } = model;
+    const useCenterOfRotation = vec3.length(model.centerOfRotation) !== 0.0;
```

Comment:
You may want to use instead `vtkMath.areEquals(model.centerOfRotation, [0, 0, 0])`